src/main/java/Deadline.java

```diff
@@ -0,0 +1,19 @@
+public class Deadline extends Task{
```

Comment:
Perhaps be more consistent with whether there's a space before the { . (This goes for many other places in the code as well.)